### DISCUSSION TITLE
Unable to log in due to academic email deactivation (verified device)

### DISCUSSION THREAD

**Discussion by jayalahd** [community/community#77250] [Other Feature Feedback, Questions, & Ideas]
Hello there,
I am unable to log into my GitHub account due to the deactivation of my previous university email. I should have added a second recovery email address but I didn't. Even though I know my password and username, the login prompts me with a TOTP authentication sent to the no longer accesible email inbox. I've tried contacting GitHub Support via the Virtual Asssistant and it gives me a few options that dont actually work.

I  have my account logged in via GitHub Desktop but have never used SSH, so I am only left with having a previously verified device with me. Since I didnt configure 2FA or any mobile verification, how can I manage to recover my account? 

I have tried to follow the steps in the https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#authenticating-with-a-verified-device-ssh-token-or-personal-access-token page but I dont have the "Having problems? >> Use a recovery codeor request a reset" option

**Comment by samus-aran** [comment]
Hi @jayalahd , Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here! 

We want to make sure you’re getting the best support you can now that GitHub Universe has ended so I've gone ahead and moved this to the most relevant category for you.

Thank you for helping us maintain a productive and tidy community for all our members.
